### ISSUE TITLE
CppCheck analyzer issue

### ISSUE THREAD

**Issue by Mayhem93** [jbenden/vscode-c-cpp-flylint#237]
Hello,

I'm having issues with this extension, I'm trying to integrate a static linter into C++ and I picked this extension + CppCheck.

VScode basically starts with this warning: 

<img width="477" height="107" alt="Image" src="https://github.com/user-attachments/assets/022cafa6-7d78-49b6-bfcb-791ca96dd34a" />

In the output of the extension I only get this:

```
Performing lint scan of d:\workspace\node-addon\src\btree_wrapper.cpp...
Completed lint scans...
Performing lint scan of d:\workspace\node-addon\src\myaddon.cpp...
Completed lint scans...
Performing lint scan of d:\workspace\node-addon\src\btree_wrapper.h...
Completed lint scans...
Performing lint scan of d:\workspace\node-addon\src\test.cpp...
Completed lint scans...
Performing lint scan of d:\workspace\node-addon\src\btree.h...
Completed lint scans...
Skipping analysis because ON_SAVE !== ON_TYPE.
```

This is the version of cppcheck (the executable is in `PATH`):

```
cppcheck --version
Cppcheck 2.16.0
```

Using windows and msvc build tools.
Extension version: `v1.15.0 `
VSCode version: `1.103.2`

This is the config of the extension:

```json
	"c-cpp-flylint.clang.enable": false,
	"c-cpp-flylint.flexelint.enable": false,
	"c-cpp-flylint.flawfinder.enable": false,
	"c-cpp-flylint.lizard.enable": false,
	"c-cpp-flylint.cppcheck.executable": " C:\\Program Files\\Cppcheck\\cppcheck.exe",
	"c-cpp-flylint.cppcheck.includePaths": [
		"C:\\Users\\uther\\AppData\\Local\\node-gyp\\Cache\\20.7.0\\include\\node"
	],
	"c-cpp-flylint.cppcheck.standard": [
		"c++20"
	]
```

I don't know if this is related but I tried to run cppcheck in cli (I added some headers -I to it, nodejs header files and everything I got installed through vpcpk) and got this in the report:

```
Critical errors
---------------
There were critical errors (syntaxError, preprocessorErrorDirective).
These cause the analysis of the file to end prematurely.
```
